core/designsystem/src/main/java/com/iteneum/designsystem/components/Text.kt

```diff
@@ -0,0 +1,205 @@
+package com.iteneum.designsystem.components
+
+import androidx.compose.foundation.clickable
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.style.TextAlign
+import com.iteneum.designsystem.theme.Boulder
+import com.iteneum.designsystem.theme.LPTypography
+
+/**
+[LPTitleLarge] is a composable function that displays a large title text with the specified label, color, and optional text alignment. It can also be clickable and trigger an onClick event.
+@param modifier Modifier is used to apply custom styles and parameters to the title, such as the width and height of the component.
+@param label The text label to be displayed as the title.
+@param color The color of the title text.
+@param textAlign Optional text alignment parameter, default is TextAlign.Center.
+@param onClick Optional high-order function that defines the click behavior when the title is clicked.
+@author Irving Ulises Gonzalez Uscanga
+
+[LPTitleMedium] is a composable function that displays a medium-sized title text with the specified label, color, and optional text alignment. It can also be clickable and trigger an onClick event.
+@param modifier Modifier is used to apply custom styles and parameters to the title, such as the width and height of the component.
+@param label The text label to be displayed as the title.
+@param color The color of the title text.
+@param textAlign Optional text alignment parameter, default is TextAlign.Center.
+@param onClick Optional high-order function that defines the click behavior when the title is clicked.
```

Comment:
Same comment here:
`I don't see the param onClick in the function, could you remove the param from the documentation if it is not need in the function or add the param onClick in the function if it need`